wpilib/wpilib/counter.py

```diff
@@ -44,7 +45,7 @@ class Counter(SensorBase):
     .. not_implemented: initCounter
     """
 
-    class Mode:
+    class Mode(IntEnum):
```

Comment:
Hm, does ctypes convert this to an integer when it goes down into HAL? Will need to remember to test that at some point.